src/pet.js

```diff
@@ -0,0 +1,79 @@
+function Pet(petName) {
+    const initialFitness = 10;
+    this.name = petName;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = initialFitness;
+    this.children = [];
+};
+
+Pet.prototype = {
+    get isAlive() {
+    const unfit = 0;
+    const starving = 10;
+    const tooOld = 30;
+    return this.age < tooOld && this.hunger < starving && this.fitness > unfit;
+}
+};
+
+Pet.prototype.growUp = function() {
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive :(');
+    }
+    this.age += 1;
+    this.hunger += 5;
+    this.fitness -= 3;
```

Comment:
Talking about the constants above, you could add these as their own constants with names such as `AGE_INCREMENT`, `HUNGER_INCREMENT` and `FITNESS_INCREMENT`